src/core/core.scale.js

```diff
@@ -857,6 +857,11 @@ module.exports = function(Chart) {
 
 					var label = itemToDraw.label;
 					if (helpers.isArray(label)) {
+						if (label.length % 2 === 0) {
+							context.translate(0, (-tickFont.size * (label.length / 2)) / 2);
+						} else {
+							context.translate(0, -tickFont.size * (label.length / 2));
+						}
```

Comment:
We need to use the line height (not only the font size) and it would be easier and faster to translate `y` instead of translating the context?

```
var lineCount = label.length;
var lineHeight = tickFont.size * 1.5;
var y = - lineHeight * lineCount / 2;
var i = 0;

for (; i < lineCount; ++i) {
  context.fillText('' + label[i], 0, y);
  y += lineHeight;
}
```